CMakeLists.txt

```diff
@@ -494,7 +494,7 @@ if(AVIF_BUILD_APPS OR (AVIF_BUILD_TESTS AND AVIF_ENABLE_GTEST))
 
     add_library(
         avif_apps STATIC apps/shared/avifjpeg.c apps/shared/iccjpeg.c apps/shared/avifpng.c apps/shared/avifutil.c
-                         apps/shared/y4m.c
+                         apps/shared/y4m.c src/diag.c src/exif.c src/stream.c src/utils.c
```

Comment:
Alternative: link to `avif_internal` from https://github.com/AOMediaCodec/libavif/pull/1185 instead of adding source files.

Reply:
This should be possible now that the change has been merged.